Args.cs

```diff
@@ -25,6 +25,9 @@ public class Args
         [Option]
         public string RelativePath { get; set; } = Environment.GetEnvironmentVariable(RelavePathEnvironmentVariable) 
                                                    ?? "/";
+        
+        [Option]
+        public int BlobTimeoutSeconds { get; set; } = int.Parse(Environment.GetEnvironmentVariable("BlobTimeoutSeconds") ?? "15");

```

Comment:
If I might suggest a more robust change: wrap this bad boy in int.Min/int.Max to set sane lower and upper bounds so misconfiguration doesn't lead to nasty surprises.

Reply:
did it in validate.